src/components/WwwFrame/TheFooter.vue

```diff
@@ -219,7 +219,7 @@ import getCacheKey from '@/util/getCacheKey';
 
 export default {
 	name: 'TheFooter',
-	serverCacheKey: () => getCacheKey('footer'),
+	serverCacheKey: props => getCacheKey(props.theme ? `footerThemed${props.theme.themeKey}` : 'footer'),
```

Comment:
When the theme is not specified within incoming props it's undefined (as opposed to picking up the default object as stated below)